src/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Schema Registry UI</title>
+    <link rel="shortcut icon" href="assets/icons/favicon.png">
+    <meta charset=utf-8/>
+    <meta name="viewport" content="width=device-width">
+    <meta name="keywords" content="kafka,schema registry,avro,landoop">
+    <meta name="description" content="Kafka Schema Registry UI">
+    <meta name="author" content="http://landoop.com">
+</head>
+
+<body ng-app="angularAPP" ng-cloak>
+
+    <!--Header Starts here-->
+    <header class="header" ng-controller="HeaderCtrl" style="background-color:{{color}}">
+        <div layout="row" layout-xs="column">
+            <div flex>
+                <span class="title"><a style="color:#fff;" class="md-padding" href="{{cluster ? '#/cluster/'+cluster.NAME : '#/'}}">SCHEMA REGISTRY</a></span>
+            </div>
+
+            <a style="" class="exportSchemas" ng-href="#/cluster/{{cluster.NAME}}/export">
+      <i class="fa fa-download" aria-hidden="true"></i> <b>EXPORT SCHEMAS</b>
+    </a>
+
+            <label ng-show="!missingEnvJS && clusters.length > 1" class="selectClusterLabel"><b>SELECT CLUSTER :</b></label>
+            <md-input-container ng-show="!missingEnvJS && clusters.length > 1" class="selectCluster">
+                <md-select ng-model="connectEndPoint">
+                    <md-option ng-repeat="connectEndPoint in clusters track by $index" ng-click="updateEndPoint(connectEndPoint.NAME)" value="{{connectEndPoint.NAME}}"
+                        ng-selected="{{cluster.NAME == connectEndPoint.NAME}}">{{connectEndPoint.NAME}}</md-option>
+                </md-select>
+            </md-input-container>
+        </div>
+    </header>
+
+    <div layout="row" flex="100" ng-show="missingEnvJS" ng-cloak>
+        <md-card>
+            <md-card-content>
+                <h3 style="font-weight:300; color:red"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Missing Cluster Configuration</h3>
+                <br> In order to configure <code>schema-registry-ui</code> you need to add <code>env.js</code> file in the
+                root directory of the app.
+                <br> Example <b>env.js</b> structure:
+                <pre>
+        <code>
+var clusters = [
```

Comment:
I didn't modify anything related to index.html, except for the imports as I don't know what features you want yet.